src/etc/inc/console.inc

```diff
@@ -383,6 +383,7 @@ EOD;
         config_read_array('interfaces', 'lan');
         $config['interfaces']['lan']['if'] = $lanif;
         $config['interfaces']['lan']['enable'] = true;
+        $config['interfaces']['lan']['antilockout'] = true;
```

Comment:
config.xml.sample needs this too, but you have to consider if people skip LAN the unfavourable (as per your view) implementation wins and keeps doing what it always did.

it might just be better to stay clear and leave the new antilockout as an opt-in for the forseeable future.

Reply:
think more about it it would make sense to simply ask the user if they want to participate in the anti-lockout group with this interface. It should be safe, because in this assignment run we assign all ports so there is no ambiguity about it.